circle.yml

```diff
@@ -1,10 +1,17 @@
-machine:
-  python:
-    version: 3.5.2
-  environment:
-    ANSIBLE_REMOTE_USER: ubuntu
-    PYTHON27: /opt/circleci/python/2.7.12/bin/python
-    TEST_HOSTS: integration/hosts
-test:
-  override:
-    - cd tests && make test
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.5.4
+        environment:
+          ANSIBLE_REMOTE_USER: ubuntu
+          TEST_HOSTS: integration/hosts
```

Comment:
You could move these settings to the command to run the tests – they are only needed for that single command, not for the full container.

Reply:
Did you see this comment?  (No need to change anything if there is any reason to leave it as it is.)

Reply:
oh you meant all of them? I thought you were only talking about the one I added "ANSIBLE_HOST_KEY_CHECKING". Sorry I will fix it.

Reply:
Specifically, I just meant to move this `environment` block below the command to run the tests, so it only applies to that command. :)